src/Misc/layoutbin/systemd.svc.sh.template

```diff
@@ -10,7 +10,10 @@ arg_2=${2}
 RUNNER_ROOT=`pwd`
 
 UNIT_PATH=/etc/systemd/system/${SVC_NAME}
-TEMPLATE_PATH=./bin/actions.runner.service.template
+TEMPLATE_PATH=$TEMPLATE_PATH
```

Comment:
Should we give it a different name for the env, maybe `$GITHUB_ACTIONS_RUNNER_SERVICE_TEMPLATE`?